### DISCUSSION TITLE
Latest 7th heaven Canary Build 3.9.9.10 missing auto-mount?

### DISCUSSION THREAD

**Discussion by elduderino-ca** [tsunamods-codes/7th-Heaven#200] [Q&A] [answered]
Hello!  I noticed, after upgrading to latest, that auto-mount disc is no longer there in launcher options.  Was this an intentional move to get rid of it, or is it a bug?  Am I the only person seeing this (something specific to my setup?)

TY!

**Comment by julianxhokaxhiu** [comment, marked as answer]
It has been removed as the next stable is going to provide native support for the Steam executable, instead of having to convert the game back to the 1998 edition. Although it will still have support for this version, with the only difference that who really owns the 1998 edition has to insert the real game disk on their drive.
